praw/models/redditors.py

```diff
@@ -9,6 +13,10 @@
 Subreddit = TypeVar("Subreddit")
 
 
+class UserSummary(SimpleNamespace):
```

Comment:
On second thought, why is this is a Namespace. Can you not subclass a PRAWBase and pass the Reddit parameter through?

Reply:
Ahh. I see the logic in that.